build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     apply from: "https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle", to: buildscript
     apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-cgp-version.gradle'
 
-    dependencies { classpath "com.synopsys.integration:common-gradle-plugin:1.12.0" }
+    dependencies { classpath "com.synopsys.integration:common-gradle-plugin:2.0.4" }
```

Comment:
@andrian-sevastyanov does this impact the recent pipeline issues/fixes you saw today?

Reply:
It's related but I don't believe there will be any impact on it.